alert.go

```diff
@@ -18,10 +18,12 @@ type Alert struct {
 	AdditionalInfo string `json:"additionalInformation"`
 
 	// Target is a comma-separated list of targets for the Alert
-	Target string `json:"target"`
+	Target  string            `json:"target"`
+	Targets map[string]string `json:"targets"`
```

Comment:
Could we just add a comment here explaining that Targets should be used for Threshold alerts? (The godocs are auto-generated from these comments)